public_html/templates/base.html

```diff
@@ -75,25 +77,39 @@
 
 	{% block main %}
 		<section class="filters container-fluid">
-			<form>
-				<fieldset class="form-group filters-fieldset text-center">
-					{% for type, description in filterTypes %}
-						<span class="radio-inline">
+			<form id="filters-form">
+				<fieldset class="form-group filters-fieldset col-lg-6">
```

Comment:
col-sm-6. We're using sm everywhere else to present it better on small screens. (in other places too)